wdl-format/tests/format/seaseq-case/source.formatted.wdl

```diff
@@ -0,0 +1,939 @@
+    }
+
+    call macs.macs as nomodel { input:
+        bamfile = sample_bam,
+        nomodel = true,
+        egs = egs.genomesize,
+        default_location = sub(basename(sample_bam), ".sorted.b.*$", "") + "/PEAKS/NARROW_peaks" + "/" + basename(sample_bam, ".bam") + "-nm",
+        coverage_location = sub(basename(sample_bam), ".sorted.b.*$", "") + "/COVERAGE_files/NARROW_peaks" + "/" + basename(sample_bam, ".bam") + "_nm",
+    }
+
+    call bamtogff.bamtogff { input:
+        gtffile = gtf,
+        chromsizes = samtools_faidx.chromsizes,
+        bamfile = select_first([
+            merge_markdup.mkdupbam,
+            mapping.mkdup_bam,
+        ]),
+        bamindex = select_first([
+            merge_mkdup.indexbam,
+            mapping.mkdup_index,
+        ]),
+        default_location = sub(basename(sample_bam), ".sorted.b.*$", "") + "/BAM_Density",
+    }
+
+    call bedtools.bamtobed as forsicerbed { input: bamfile = select_first([
+        merge_markdup.mkdupbam,
+        mapping.mkdup_bam,
+    ]) }
+
+    call sicer.sicer { input:
+        bedfile = forsicerbed.bedfile,
+        chromsizes = samtools_faidx.chromsizes,
+        genome_fraction = egs.genomefraction,
+        fragmentlength = select_first([
+            uno_bfs.readlength,
+            mergebam.avg_readlength,
+        ]),
+        default_location = sub(basename(sample_bam), ".sorted.b.*$", "") + "/PEAKS/BROAD_peaks",
+        coverage_location = sub(basename(sample_bam), ".sorted.b.*$", "") + "/COVERAGE_files/BROAD_peaks",
+    }
+
+    call rose.rose { input:
+        gtffile = gtf,
+        bamfile = select_first([
+            merge_markdup.mkdupbam,
+            mapping.mkdup_bam,
+        ]),
+        bamindex = select_first([
+            merge_mkdup.indexbam,
+            mapping.mkdup_index,
+        ]),
+        bedfile_auto = macs.peakbedfile,
+        bedfile_all = all.peakbedfile,
+        default_location = sub(basename(sample_bam), ".sorted.b.*$", "") + "/PEAKS/STITCHED_peaks",
+    }
+
+    call runspp.runspp { input: bamfile = sample_bam }
+
+    call util.peaksanno { input:
+        gtffile = gtf,
+        bedfile = macs.peakbedfile,
+        chromsizes = samtools_faidx.chromsizes,
+        summitfile = macs.summitsfile,
+        default_location = sub(basename(sample_bam), ".sorted.b.*$", "") + "/PEAKS_Annotation/NARROW_peaks" + "/" + sub(basename(macs.peakbedfile), "_peaks.bed", ""),
+    }
+
+    call util.peaksanno as all_peaksanno { input:
+        gtffile = gtf,
+        bedfile = all.peakbedfile,
+        chromsizes = samtools_faidx.chromsizes,
+        summitfile = all.summitsfile,
+        default_location = sub(basename(sample_bam), ".sorted.b.*$", "") + "/PEAKS_Annotation/NARROW_peaks" + "/" + sub(basename(all.peakbedfile), "_peaks.bed", ""),
+    }
+
+    call util.peaksanno as nomodel_peaksanno { input:
+        gtffile = gtf,
```

Comment:
Not for this PR, just a general comment. I really dislike seeing `gtffile` here. I wonder if we should expand `disallowed_input_name` to disallow `file`, `int`, etc. prefixes and suffixes. Since the inputs are typed, I don't think we need the type rewritten in the name.

Reply:
I 100% agree. Going to open an issue to add that. Although, this is a pretty controversial lint of ours. I'm ok with it, but just pointing out it's probably going to annoy some people.